### PR TITLE
Cleanup Installer plugins

### PR DESCRIPTION
**Pull request by viocassel**
Pull Request for Issue # .

### Summary of Changes
🤪 Just some small cleanup... 

### Testing Instructions
Check if the extension is installed correctly using the drag and drop installer.

### Expected result
The installer is working correctly.

### Actual result
The installer is working correctly.

### Documentation Changes Required
No

### CODE REVIEW

File: plugins/installer/folderinstaller/tmpl/default.php [hunk 1]
```diff
@@ -14,33 +14,27 @@
...
+			form.installtype.value = "folder";
 			form.submit();
 		}
 	};
 ');
 ?>
 <legend><?php echo JText::_('PLG_INSTALLER_FOLDERINSTALLER_TEXT'); ?></legend>
 <div class="control-group">
-	<label for="install_directory" class="control-label"><?php echo JText::_('PLG_INSTALLER_FOLDERINSTALLER_TEXT'); ?></label>
+	<label class="control-label" for="install_directory"><?php echo JText::_('PLG_INSTALLER_FOLDERINSTALLER_TEXT'); ?></label>
```
Comment (brianteeman) [inline plugins/installer/folderinstaller/tmpl/default.php]: personally I prefer `label for=` for the same reason that id goes before styling 
does the style guide say anything about this?

Comment (viocassel) [reply plugins/installer/folderinstaller/tmpl/default.php]: Ok :)

**Comment by jwaisner** [comment]
I have tested this item :white_check_mark: successfully on 057b2174ebba19f63263f6737b4ea24341104d1f<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27123">issues.joomla.org/tracker/joomla-cms/27123</a>.</sub>

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on 057b2174ebba19f63263f6737b4ea24341104d1f<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27123">issues.joomla.org/tracker/joomla-cms/27123</a>.</sub>

**Comment by Quy** [comment]
Please fix conflicting files.

**Comment by Quy** [comment]
@zero-24 This PR is mainly coding style changes. Since the branch has been deleted, it is worth redoing? Should this be closed?

**Comment by zero-24** [comment]
Please do not understand me wrong but for now I would say when we have to redo it anyway target such cleanups against 4.1/4.2 rather than 3.10. Sure when we have topics like the php 8.1 adoption I'm happy to take it into 3.10 but such general cleanups often result into messy merge conflicts when merging 3.10 into 4.x. For the JS changes I'm not sure whether they are acutuall bug fixes when they are we should take them and target them against 3.10.

**Comment by laoneo** [comment]
As stated above, the pr should be done against 4.2.

**Pull request closed without merging**